data/warlock/document.json

```diff
@@ -0,0 +1,245 @@
+            "7. Use of Product Identity: You agree not to Use any Product Identity, including as an indication as to compatibility, except as expressly licensed in another, independent Agreement with the owner of each element of that Product Identity. You agree not to indicate compatibility or co-adaptability with any Trademark or Registered Trademark in conjunction with a work containing Open Game Content except as expressly licensed in another, independent Agreement with the owner of such Trademark or Registered Trademark. The use of any Product Identity in Open Game Content does not constitute a challenge to the ownership of that Product Identity. The owner of any Product Identity used in Open Game Content shall retain all rights, title and interest in and to that Product Identity.",
+            "8. Identification: If you distribute Open Game Content You must clearly indicate which portions of the work that you are distributing are Open Game Content.",
+            "9. Updating the License: Wizards or its designated Agents may publish updated versions of this License. You may use any authorized version of this License to copy, modify and distribute any Open Game Content originally distributed under any version of this License.",
+            "10. Copy of this License: You MUST include a copy of this License with every copy of the Open Game Content You Distribute.",
+            "11. Use of Contributor Credits: You may not market or advertise the Open Game Content using the name of any Contributor unless You have written permission from the Contributor to do so.",
+            "12. Inability to Comply: If it is impossible for You to comply with any of the terms of this License with respect to some or all of the Open Game Content due to statute, judicial order, or governmental regulation then You may not Use any Open Game Material so affected.",
+            "13. Termination: This License will terminate automatically if You fail to comply with all terms herein and fail to cure such breach within 30 days of becoming aware of the breach. All sublicenses shall survive the termination of this License.",
+            "14. Reformation: If any provision of this License is held to be unenforceable, such provision shall be reformed only to the extent necessary to make it enforceable.",
+            "15. COPYRIGHT NOTICE",
+            "Open Game License v 1.0a Copyright 2000, Wizards of the Coast, Inc.",
+            "12 Perilous Towers © 2018 Open Design LLC; Authors: Jeff Lee.",
+            "A Drinking Problem ©2020 Open Design LLC. Author Jonathan Miley.",
+            "A Leeward Shore Author: Mike Welham. © 2018 Open Design LLC.",
```

Comment:
I think we should slim down these source docs to only the ones that are relevant, eg. The various Warlock publications for this set?

Happy to hop on this myself this morning, if you'd like.

Reply:
I agree, but that can be done transparently after-the-fact, so it wasn't a big priority for me.